src/nekontrol/interactive/cli.py

```diff
@@ -16,26 +15,24 @@ def cli():
     ...
 
 
-@cli.command(context_settings={"help_option_names": ["-h", "--help"]})
+@cli.command("test", context_settings={"help_option_names": ["-h", "--help"]})
 @click.argument("file-path", metavar="FILE", type=executable_file)
 @click.option("-p", "--problem", type=str, help="The kattis problem name")
 @click.option("-d", "--diff", type=bool)
 @click.option("-c", "--color", type=bool)
 @click.option("-v", "--verbose", type=bool)
 @click.option("--ignore-debug", type=bool)
-def test(
+def test_cli(
     file_path: str,
-    problem: str | None,
-    color: bool | None,
-    diff: bool | None,
-    ignore_debug: bool | None,
-    verbose: bool | None,
+    problem: Optional[str],
+    color: Optional[bool],
+    diff: Optional[bool],
+    ignore_debug: Optional[bool],
+    verbose: Optional[bool],
```

Comment:
`Optional[t]` was primarily used as a shorthand for `Union[t, None]`, in newer python versions, since the pipe was introduced, `t | None` is most common, so I would prefer that.